src/apps/material/login-redirect.test.ts

```diff
@@ -0,0 +1,67 @@
+const coverUrlPattern = /^https:\/\/res\.cloudinary\.com\/.*\.(jpg|jpeg|png)$/;
+
+describe("Material", () => {
+  it("Redirects to login & opens reservation modal on subsequent land-in", () => {
+    cy.visit("/iframe.html?id=apps-material--default&type=bog")
+      .getBySel("material-header-buttons-physical")
+      .click();
+
+    cy.interceptRest({
+      aliasName: "user",
+      url: "**/agencyid/patrons/patronid/v2",
+      fixtureFilePath: "material/user.json"
+    });
+
+    cy.interceptRest({
+      aliasName: "holdings",
+      url: "**/agencyid/catalog/holdings/**",
+      fixtureFilePath: "material/holdings.json"
+    });
+
+    cy.url().should("include", "modal=reservation-modal-46615743");
+    cy.getBySel("modal").should("be.visible");
+  });
```

Comment:
Fails locally both when running:
`yarn test` and `yarn cypress:open`
Does it work in GH actions?

Reply:
Maybe we need a way to intercept the redirect to Adgangsplatformen?

Reply:
mmmmm works just fine for me
![image](https://user-images.githubusercontent.com/28546954/214274329-401e65ae-6005-4baf-802b-1cbc1bd95930.png)

As far as I know the setup for testing is that you need to run `yarn start:storybook:test` and then `yarn cypress:open`

Reply:
Det fungerer på GH også:
![image](https://user-images.githubusercontent.com/28546954/214274776-f3c65106-d387-4d36-9e45-54a2703801be.png)

Reply:
I think it could be the user token still present before pressing the reserve button (in which case no redirect happens). I made some changes and you can try again 🤞